util/build_docs.py

```diff
@@ -240,6 +252,11 @@ def main():
     os.environ["PATH"] += os.pathsep + str(hugo_localinstall_dir)
 
     try:
+        if not is_hugo_extended():
+            logging.info("Hugo version is not extended. "
```

Comment:
wording nit: "Hugo extended (with SASS support) is required, but only non-extended version found. Installing local copy and re-trying."